lib/pure/math.nim

```diff
@@ -1131,3 +1133,24 @@ func lcm*[T](x: openArray[T]): T {.since: (1, 1).} =
   while i < x.len:
     result = lcm(result, x[i])
     inc(i)
+
+
+when not defined(js) and not defined(nimscript):
+  func copySign*[T: SomeFloat](x, y: T): T {.inline, since: (1, 5, 1).} =
```

Comment:
for js how about:
```
https://www.30secondsofcode.org/js/s/copy-sign
```

Reply:
Yeah, it could be considered BUT it makes C and JS backend inconsistent.

Becuase JS is hard to implement  signbits. And copySign is based on signbits instead of sign function.

Reply:
ok, can be done in future work